src/libmongoc/src/mongoc/mongoc-server-api.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mongoc-prelude.h"
+
+#ifndef MONGOC_SERVER_API_H
+#define MONGOC_SERVER_API_H
+
+#include <bson/bson.h>
+
+#include "mongoc-macros.h"
+
+BSON_BEGIN_DECLS
+
+typedef enum { MONGOC_SERVER_API_V1 = 1 } mongoc_server_api_version_t;
```

Comment:
Do we manually need to assign values to the enum? IMO, this could sow confusion whether versions are strings or numbers.

Reply:
No, I suppose we don't need to do this, now that I've changed the to string logic to not rely on the numeric value of the enum.  Will change.